services/api/src/admin_user.py

```diff
@@ -48,11 +67,19 @@ def get_user_data(user_email):
         return user_list
 
 
-def get_modify_user_data(user_email):
+@require_permission(
+    required_role=ORG_ROLE_LITERAL.ADMIN,
+    resource_type=RESOURCE_TYPE.USER,
+)
+def get_modify_user_data_authorized(
+    user_email: str, permission_result: PermissionResult
+):
     modify_user_obj = {}
-    modify_user_obj["user_data"] = get_user_data(user_email)
+    modify_user_obj["user_data"] = get_user_data(user_email, permission_result)
```

Comment:
chore: provide all expected inputs to method call. this may not be the correct third input, but it seems to work
```suggestion
    modify_user_obj["user_data"] = get_user_data(user_email, permission_result, [])
```